### ISSUE TITLE
WP rewrites

### ISSUE THREAD

**Issue by juliotaylor** [bigspring/monolith#62]
- [ ] Implement a rewrite so all images and assets are not in wp_uploads
- [ ] change wp-admin path to conceal all WP admin traces
- [ ] hide all references to wp-content

**Comment by juliotaylor** [comment]
@jrgmartin I'd like this in 2.1 – is it doable?

**Comment by jrgmartin** [comment]
Yep, totally do-able.  I expect it'll be quite tricky to setup so will need a good half day at least to get it right.
